msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -190,8 +190,9 @@ void acquireToken(@NonNull final Activity activity,
          * @param vUri verification uri
          * @param userCode user code
          * @param message instruction message
+         * @param expiresIn the number of seconds before the DCF session expires.
```

Comment:
This is a really unfortunate value.  I would prefer to see the UTC time that the DCF session expires at, because that signal, while subject to clock skew, would at least not degrade over the time that the system waits before getting around to displaying it.

But if this could be a Duration (number of seconds) or an Instant (precise time point) at least we would help people writing calls to this method sort out their parameters.

Reply:
That's a good idea. Let me do that.

(Note that I did call out that they should be using this for displaying purpose only - not to use it to decide when they should retry - we're already returning a specific exception and they should use that as a signal instead . In that sense - a slight delay should be acceptable. I'll also add a comment to stress on that too.